using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -36,6 +40,12 @@ public class CoreApp extends Application {
 {% endtab %}
 {% endtabs %}
 
+
+
+### Get Application
+
+You can use this API to get the previous set Android `Application` instance and it is mainly provided for the third-party extensions.
```

Comment:
Please reword as follows:
"You can use this API to get the previously set Android `Application` instance, and this instance is mainly provided for the third-party extensions."